src/Alert/Alert.stories.js

```diff
@@ -0,0 +1,194 @@
+// @flow
+import * as React from "react";
+import { storiesOf, setAddon } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import styles from "@sambego/storybook-styles";
+import chaptersAddon from "react-storybook-addon-chapters";
+import { withKnobs, text, boolean, select } from "@storybook/addon-knobs/react";
+
+import ThemeProvider from "../Theming/ThemeProvider";
+import * as Icons from "../icons";
+
+import Alert from "./index";
+
+setAddon(chaptersAddon);
+
+const getIcons = defaultIcon => select("Icon", [undefined, ...Object.keys(Icons)], defaultIcon);
+const getIcon = source => Icons[source];
+
```

Comment:
we can add `renderTitle` fci here

```
const renderTitle = (title) => title !== "" && title
```

to prevent repeating code

then use:

```
<Alert title={renderTitle(title)} />
```

Reply:
Will fix it.